docs/formats/avro.md

```diff
@@ -188,13 +189,13 @@ The following properties can be used to configure the Avro encoder. All of these
 | `namespace`                    | string | | Avro namespace for the generated Avro schemas.|
-| `subject_name_strategy`        | `"topic_name"`, `"record_name"`, or `"topic_record_name"` | see description| Subject name strategy used to publish Avro schemas used by the connector in the schema registry. When this property is not specified, the connector chooses subject name strategy automatically, `topic_name` for `confluent_jdbc` update format or `record_name` for `raw` update format.|
-| `key_fields`                   | array of strings| | When this option is set, only the listed fields appear in the Debezium message key. This option is only valid with the `confluent_jdbc` update format. It is used when writing to a table with primary keys. For such tables, the Confluent JDBC sink connector expects the message key to contain only the primary key columns. When this field is set, the connector generates a separate Avro schema, containing only the listed fields, and uses this schema to encode Kafka message keys.|
+| `subject_name_strategy`        | `"topic_name"`, `"record_name"`, or `"topic_record_name"` | `topic_name` for `confluent_jdbc` update format or `record_name` for `raw` update format | <p>Subject name strategy used to publish Avro schemas used by the connector in the schema registry.</p><p>`topic_name`: the subject name is derived from the Kafka topic name. For update formats with both key and value components, use subject names `{topic_name}-key` and `{topic_name}-value` for key and value schemas respectively. For update formats without a key (e.g., `raw`, with `key_mode=none`), publish value schema under the subject name `{topic_name}`. Only applicable when using Kafka as a transport.</p><p>`record_name`: the name of the SQL relation name that the schema is derived from is used as the subject name: the SQL view name for the message value schema or the SQL index name for the message key schema.</p><p>`topic_record_name`: combines both the topic name and the record name to form the subject. For update formats with both key and value components, use subject names `{topic_name}-{record_name}-key` and `{topic_name}-{record_name}-value` for key and value schemas respectively. For update formats without a key, publish value schema under the subject name `{topic_name}-{record_name}`. Only applicable when using Kafka as a transport.</p>|
```

Comment:
maybe the last should be called `topic_and_record`

Reply:
I did not invent this. These are standard names in the Avro world.